frontend/packages/console-app/console-extensions.json

```
@@ -425,6 +425,22 @@
       "insertAfter": "topology.preferredView"
     }
   },
+  {
+    "type": "console.user-preference/item",
+    "properties": {
+      "id": "console.enableExactSearch",
+      "label": "%console-app~Search%",
+      "groupId": "general",
+      "description": "%console-app~Enable/disable exact string match when filtering on resources.%",
```

I would leave out the `disable` part
```suggestion
      "description": "%console-app~Enable exact string match filtering on resources.%",
```

but not a native speaker so I would rather defer to @opayne1

---

I'd say remove disable to stay consistent with the rest of the PR. We also should [refrain from using "/"](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-slashes) because it can be ambiguous. Thanks for looping me in!

---

@jhadvig @opayne1 removed the /disable from the description text.